osu.Game.Rulesets.Osu/Difficulty/Skills/Speed.cs

```diff
@@ -20,20 +23,116 @@ public class Speed : Skill
         private const double pi_over_4 = Math.PI / 4;
         private const double pi_over_2 = Math.PI / 2;
 
+        private const double streamaimconst = 2.42;
+
+        //  private const double stdevconst = 0.149820;
+
+        private int sdsplitcounter = 0;
+        private int sdsplitcounter2 = 0;
+        // private int runcheck1 = 0;
+
+        private double sdstrainmult = 0;
+        // private double sdstrainmult2;
+
+        // private double sdstrainmult2;
+
+        public readonly List<double> JumpDistances = new List<double>();
+        public readonly List<double> StrainTimes = new List<double>();
+        public List<double> JumpDistances2 = new List<double>();
```

Comment:
Why is there `JumpDistances` and `JumpDistances2`? Why are they public? xmldoc please.

Reply:
Will change to private, was public because I need to move variable across files but now it is unnecessary. I'll also explain using comments too.

Reply:
Fixed in 55eaa82.